src/cata_utility.h

```diff
@@ -40,7 +40,9 @@ double logarithmic( double t );
 double logarithmic_range( int min, int max, int pos );
 
 int bound_mod_to_vals( int val, int mod, int max, int min );
+std::string velocity_units( bool to_alternative_units = false );
```

Comment:
@BevapDin I like your idea about enum.

Since you're here, I want to ask you about better output format for ```*_units``` functions. Currently, return value is ```std::string```, and all current calls of named functions ends with ```.c_str()```. Should I try to change return value to ```char``` or something else or it's okay to keep everything as is?